src/main/java/net/sf/jabref/importer/fetcher/ACMPortalFetcher.java

```diff
@@ -452,6 +449,7 @@ private String getResults(URL source) throws IOException {
                 sb.append((char) buffer[i]);
             }
         }
+        in.close();
```

Comment:
Yes, there are a lot of cases. But not every case is easily translated into try-with-resources. Especially when the opening and closing of the resource is split into completely different locations and points in the execution flow. 

With try-with-resources, you do not need to call `close` as this construct does this for you. 

I would do this always then the resource I am using implements the `AutoCloseable` interface. As you never know. :) But if a `Scanner` would only work on a `String`, this probably must not be closed. But I am not certain in that case.